lib/widgets/lightbox.dart

```diff
@@ -492,11 +493,25 @@ class _VideoLightboxPageState extends State<VideoLightboxPage> with PerAccountSt
     _controller?.removeListener(_handleVideoControllerUpdate);
     _controller?.dispose();
     _controller = null;
+    // The VideoController doesn't emit a pause event
+    // while disposing, so disable the wakelock here
+    // explicitly.
+    ZulipBinding.instance.toggleWakelock(enable: false);
```

Comment:
Good to have caught this — this API seems a bit fragile.

It would also get quite buggy if we ever had several widgets asking for this at the same time; for example, the first one that went away would turn it off even though the others were still around.

A good, fully Flutter-style API for this would probably involve a widget whose job is to keep the screen on for whatever window it appears in, and that behind the scenes deals with talking to the plugin. But that is a problem for an ambitious maintainer of `package:wakelock_plus` (or author of an alternative package) to solve, not us :slightly_smiling_face:. I think we'll only have one of these active at a time.